supplements/xanathars-guide-to-everything/feats.xml

```diff
@@ -4,10 +4,11 @@
 		<name>Feats</name>
 		<description>The Feats from Xanathar’s Guide to Everything</description>
 		<author url="http://dnd.wizards.com/products/tabletop-games/rpg-products/xanathars-guide-everything">Xanathar’s Guide to Everything</author>
-		<update version="0.0.7">
+		<update version="0.0.8">
 			<file name="feats.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/xanathars-guide-to-everything/feats.xml" />
 		</update>
 	</info>
+
 	<element name="Bountiful Luck" type="Feat" source="Xanathar’s Guide to Everything" id="ID_WOTC_XGTE_FEAT_BOUNTIFUL_LUCK">
 		<prerequisite>Halfling</prerequisite>
 		<requirements>ID_RACE_HALFLING||ID_WOTC_TCOE_RACE_CUSTOM_LINEAGE||ID_INTERNAL_GRANT_RACE_HALFLING</requirements>
```

Comment:
With the changes to Custom Lineage we should also remove the "||ID_WOTC_TCOE_RACE_CUSTOM_LINEAGE" from all race feat requirements.   It is technically a breaking change but we can also make items to grant the new grants also used in the requirements if needed